SC_redesign/Crop_and_Soil/field/manure_application.py

```diff
@@ -204,6 +235,10 @@ def _apply_liquid_machine_manure(self, dry_matter_mass: float, dry_matter_fracti
             Total mass of phosphorus in this application of manure (kg)
         field_coverage : float
             Fraction of the field this manure is applied to (unitless)
```

Comment:
A few others here missing periods.
```suggestion
            Fraction of the field this manure is applied to (unitless).
```